### PR TITLE
bulk_deletion_send_to_es

### PR DESCRIPTION
**Pull request by ypersky1980**
This PR implements sending test_multiple_pvc_deletion_measurement_performance ( which measures bulk deletion time) results to the ES for later upload to the Performance Dashboard. 

Signed-off-by: Yulia Persky <ypersky@redhat.com>

### CODE REVIEW

File: tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py [hunk 1]
```diff
@@ -489,3 +489,30 @@ def test_multiple_pvc_deletion_measurement_performance(self, teardown_factory):
...
+        full_results.add_key("interface", self.interface)
+        full_results.add_key("bulk_size", number_of_pvcs)
+        full_results.add_key("pvc_size", pvc_size)
+        full_results.all_results["bulk_deletion_time"] = pvc_deletion_time
+
+        if self.full_results.es_write():
+            res_link = self.full_results.results_link()
+            log.info(f"The Result can be found at : {res_link}")
+
+            # Create text file with results of all subtest (4 - according to the parameters)
```
Comment (Avilir) [inline tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py:517]: please update the comment to 3 - you have only 3 tests in this parametrize and not 4.

File: tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py [hunk 2]
```diff
@@ -282,6 +282,7 @@ def test_pvc_creation_deletion_measurement_performance(
             # Create text file with results of all subtest (4 - according to the parameters)
             self.write_result_to_file(res_link)
 
+    @pytest.mark.skip(SKIP_REASON)
```
Comment (Avilir) [inline tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py:285]: why skipping this part ?

File: tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py [hunk 3]
```diff
@@ -489,3 +490,54 @@ def test_multiple_pvc_deletion_measurement_performance(self, teardown_factory):
...
+        This is not a test - it is only check that previous test ran and finish as expected
+        and reporting the full results (links in the ES) of previous tests (3)
+        """
+        self.number_of_tests = 3
+        results_path = get_full_test_logs_path(
+            cname=self, fname="test_multiple_pvc_deletion_measurement_performance"
+        )
+        self.results_file = os.path.join(results_path, "all_results.txt")
+        log.info(f"Check results in {self.results_file}.")
+        log.info("Check results for 'performance_extended' marker (3 tests)")
```
Comment (Avilir) [inline tests/e2e/performance/csi_tests/test_pvc_creation_deletion_performance.py:540]: this line can be removed.

**Review by Avilir — APPROVED** [review]
LGTM

**Comment by keemano** [comment]
LGTM

**Review by pintojoy — APPROVED** [review]
LGTM

**Pull request merged**